provider/testdata/data/bincode/manifest.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Issue: Please remove these `provider/testdata/data/bincode` files from the PR.

Reply:
Whoa. I wonder what git problem caused these. Sorry.

Reply:
Now removed from the PR.